templates/common/custom_filter.conf.epp

```diff
@@ -1,6 +1,11 @@
 # Fail2ban custom filter
 # Managed by Puppet
 
+[Init]
+<% if $journalmatch { -%>
```

Comment:
Shouldn't we include the `[Init]` within the `if` statement? And is the first `I` really capitalized?

Reply:
The "I" is capitalized in the files that come in the RPM. The `[Init]` can contain more than $journalmatch, so probably shouldn't be tied together. There should be no issues with having an empty section in the config in case $journalmatch isn't defined.